src/AccessibilityInsights.SharedUx/Controls/ScannerResultControl.xaml.cs

```diff
@@ -37,6 +37,7 @@ public ScannerResultControl()
             InitializeComponent();
             _list = new List<ScanListViewItemViewModel>();
             Resources.Source = new Uri(@"pack://application:,,,/AccessibilityInsights.SharedUx;component/Resources/Styles.xaml", UriKind.Absolute);
+            spHowToFix.DataContext = null;
```

Comment:
This one also seems odd. The debugger says that `spHowToFix.DataContext` is already null so this shouldn't change anything. The warning refers to not finding `MainWindow.HowToFixText`, which doesn't exist because the `HowToFixText` property is on `ScanListViewItemViewModel`. I'm not sure how to fix the context mismatch on this--maybe we're missing a `ScannerResultControlViewModel` ?

Reply:
Yeah, I wasn't sure how to address this either. I ended up going with this because of the logic in the [UpdateTree method](https://github.com/microsoft/accessibility-insights-windows/blob/main/src/AccessibilityInsights.SharedUx/Controls/ScannerResultControl.xaml.cs#L111). My assumption is that we reassign the datacontext there when necessary, and the default in that method if for the datacontext to be null. As a result, I figured it would make sense to start by explicitly initializing it as null too, which will only last until UpdateTree is first called anyways. I'm definitely open to other suggestions though!

Reply:
Yeah, looking [here](https://github.com/microsoft/accessibility-insights-windows/blob/main/src/AccessibilityInsights.SharedUx/Controls/ScannerResultControl.xaml.cs#LL217C14-L217C14) we use null as a valid value. Let's go with this, even though it's a bit odd